### PR TITLE
show fullname, username, date in proposal list & detail view

### LINKED ISSUE #71: Add Proposed date in proposal list and detail page
Currently Proposal Type, section, created by is displayed in list page. It is good to have date proposed in list page. Same applies to detail page.

Listurl: <conference>/proposals/
Detail: <conference>/proposals/<proposal-title>/

### PR DESCRIPTION
**Pull request by ChillarAnand**
resolves #71, #73 

LIST VIEW:

DETAIL VIEW:

### CODE REVIEW

File: junction/templates/proposals/detail/base.html [hunk 1]
```diff
@@ -27,10 +27,8 @@
                 {{ proposal.title }}
             </h1>
             <span>
-            by:
-            <b>
-            {{ proposal.author.get_full_name }}
-            </b>
+                by: <b>{{ proposal.author }} ({{ proposal.author.get_full_name }})</b>
```
Comment (theskumar) [inline junction/templates/proposals/detail/base.html]: Let's have it like `{{ proposal.author.username }}` to be more explicit and relying less on default string representation of `User` object.

File: junction/templates/proposals/list.html [hunk 2]
```diff
@@ -121,7 +121,8 @@ <h2 class="title-underline">Proposal Types</h2>
                             <span class="pull-right">
                                 <b>Proposal Type:</b> {{ proposal.proposal_type }}
                                 <b>Proposal Section:</b> {{ proposal.proposal_section }}
-                                <b>By:</b> {{ proposal.author }}
+                                <b>By:</b> {{ proposal.author }} ({{ proposal.author.get_full_name }})
```
Comment (theskumar) [inline junction/templates/proposals/list.html]: same here. can we use `{{ proposal.author.username }}`

**Comment by theskumar** [comment]
Can we remove `time` from the datetime format? It adds extra info without any significant constructive usages.

**Comment by ChillarAnand** [comment]
t1


t2

**Comment by theskumar** [comment]
Looks great! Merging this in!

**Pull request merged**